js-exercises/abbreviate-string/abbreviateString.test.js

```diff
@@ -0,0 +1,16 @@
+import { abbreviateString } from './abbreviateString';
+
+describe('abbreviateString', () => {
+  it('abbreviates the following strings', () => {
+    expect(abbreviateString('Hacktober Fest')).toEqual('Hacktober F.');
+    expect(abbreviateString('Leeroy Fitzgerald Jenkins')).toEqual('Leeroy J.');
+    expect(abbreviateString('Some arbitrary string length here.')).toEqual('Some H.');
```

Comment:
I think if you had written an extra test, you'd have thought of few other edge cases too :)